### ISSUE TITLE
How to check home assistant behind nginx ssl proxy? (uptime kuma ha addon)

### ISSUE THREAD

**Issue by DjordjeMandic** [louislam/uptime-kuma#3815]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

I would like to check if HA is reachable via nginx ssl proxy.

The problem is that I don't have NAT loopback. When using domain I get timeout because it cant connect to my external ip. I have adguard rewriting dns request but uptime kuma is not using adguard for dns and I kept geting ssl errors. Then I went with approach to put ip instead of domain and check "ignore tls/ssl errors". That fixed ssl error issue but I now get "socket hang up".

I tried using HTTP(s) monitor type and left most as default.

Basically I want to check if I'm getting HA home page or nginx 502 bad gateway.

### 🐻 Uptime-Kuma Version

1.23.1

### 💻 Operating System and Arch

Home Assistant OS

### 🌐 Browser

Google Chrome 117.0.5938.92 (Official Build) (64-bit)

**Comment by CommanderStorm** [comment]
See https://github.com/louislam/uptime-kuma/wiki/Troubleshooting

Your problem is not quite clear to me based on your description.
Could you please draw a network map and annotate where you tested via curl?

**Comment by CommanderStorm** [comment]
Assuming resovled as @DjordjeMandic did not respond
